traffic_portal/app/src/common/modules/table/serverDeliveryServices/TableServerDeliveryServicesController.js

```diff
@@ -82,7 +82,7 @@ var TableServerDeliveryServicesController = function(server, deliveryServices, $
 	$scope.cloneDsAssignments = function(server) {
 		var params = {
 			title: 'Clone Delivery Service Assignments',
-			message: "Please select an " + server.type + " cache to assign these " + deliveryServices.length + " delivery services to.<br><br>Warning - Any delivery services currently assigned to the target cache will be lost and replaced with these delivery service assignments...",
+			message: "Please select another " + server.type + " cache to assign these " + deliveryServices.length + " delivery services to.<br><br><strong>WARNING THIS CANNOT BE UNDONE</strong> - Any delivery services currently assigned to the selected cache will be lost and replaced with these " + deliveryServices.length + " delivery service assignments.",
```

Comment:
This line is pretty long, I'm not sure if webpack will let you but it might be worth trying to make this a template string:
```javascript
`Please select another ${server.type} cache to assign these ${deliveryServices.length} delivery services to.
<br/>
<br/>
<strong>WARNING THIS CANNOT BE UNDONE</strong> - Any delivery services currently assigned to the selected cache will be lost and replaced with these ${deliveryServices.length} delivery service assignments.`
```
This really isn't how DOM manipulation is supposed to be done with Angular, but I suppose it's too late to complain about `$uibModal` now.

Reply:
I guess webpack can't handle the modern world :(